cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +226,81 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message RegisterFileRequest {
+    Identifier survey = 1;                                          // [required] Either name or id of a survey previously registered with /RegisterSurvey
+    string path = 2;                                                // [required if file is not synthetic] Path including protocol, bucket and directory structure. Example: "gs://cognite-seismic-eu/samples/"
```

Comment:
I am wondering if we should have the full cloud storage URL in this place instead. It's kinda nice to have a name though, but honestly that's the externalid... Hmm!

Reply:
What's the difference between the 'path' and the 'full cloud storage url'?

Reply:
I guess 'full url' = 'path' / 'name'